scripts/wildcard-domain-processor/top-tld.ts

```diff
@@ -0,0 +1,205 @@
+/**
+ * List of top level domains
+ */
+export const TOP_LEVEL_DOMAIN_LIST = [
```

Comment:
TLDs are listed in http://data.iana.org/TLD/tlds-alpha-by-domain.txt

You may can use a package, like https://www.npmjs.com/package/global-tld-list

Reply:
> Currently, they're just converted to a list using top 200 most popular TLD.
this is from the task
and we will allow filter developers to update this list

Reply:
Let's add a comment that explains that we're operating a limited list of TLDs, otherwise the script will work for hours.

Also, if this list is changed, it is required to re-run the script that expands wildcards.